StreetHawk/Classes/Core/Publish/SHUtils.m

```diff
@@ -96,6 +96,18 @@ void SHLog(NSString *format, ...)
     return [shGetDateFormatter(@"yyyy-MM-dd'T'HH:mm:ssZ", nil, nil) stringFromDate:date];
 }
 
+NSDate *getCurrentLocalDateTime()
+{
+    NSDate* sourceDate = [NSDate date];
```

Comment:
NSDate is "local" by itself. It has timezone by itself. For example, one NSDate is 2018-1-1 12:00:00 in Sydney timezone, and the same NSDate is 2018-1-1 10:00:00 in Shanghai timezone. 
You are doing is like "formatting" NSDate in certain timezone, making it looks like a string in a certain timezone.